src/main/resources/org/jenkinsci/plugins/authorizeproject/strategy/SpecificUsersAuthorizationStrategy/checkPasswordRequested.js

```diff
@@ -74,8 +74,10 @@ Behaviour.specify(".specific-user-authorization", "checkPasswordRequired", 0, fu
   
   var onchange = function(evt) {
     var url = useridField.getAttribute("checkPasswordRequestedUrl");
-    url = eval(url);
-    fetch(url, {
+    var params = new URLSearchParams({

```

Comment:
A while back we had problems with `URLSearchParams` not working in HtmlUnit, so I would feel more comfortable with a PCT run to make sure this won't cause unnecessary hassle in test coverage. An ATH run wouldn't hurt too, but I won't insist on it.

Reply:
PCT run:

* https://github.com/jenkinsci/bom/pull/3798

ATH run:

* https://github.com/jenkinsci/acceptance-test-harness/pull/1794